components/inference-service/requirements.md

```diff
@@ -0,0 +1,51 @@
+## Overview
+As data scientists & applications developers build AI models they need a way to deploy these applications in production. MONAI Inference Service will be used to deploy a MONAI application. This proposal documents the requirements for the MONAI Inference Service (MIS).
+
+## Goal
+The goal for this proposal is to enlist, prioritize and provide clarity on the requirements for MIS. Developers working on different software modules in MIS SHALL use this specification as a guideline when designing and implementing software for the Service.
+
+## Success Criteria
+MIS SHALL enable clients to communicate with it via a REST based RPC API.
+
+Clients SHALL be able to register the configuration of a [MONAI Application Packages(MAP)](https://github.com/Project-MONAI/monai-deploy/blob/main/guidelines/monai-application-package.md#manifests) with MIS. 
```

Comment:
I do think any of the requirements for MIS should start with `Clients SHALL`. These are requirements for MIS, not for clients.

maybe `MIS SHALL support configuration of the MONAI Application Package (MAP) used to service inference requests.`?